.env_sample

```diff
@@ -1,7 +1,7 @@
 # Development options
 
 # App core values
-APP_DOMAIN="localhost:3000"
+APP_DOMAIN="forem.test"
```

Comment:
On GH Actions, some tests are still failing with invalid domain.
```
1st Try error in ./spec/system/feedback_message_spec.rb:13:
invalid domain: ".localhost:3000"

RSpec::Retry: 2nd try ./spec/system/feedback_message_spec.rb:13

2nd Try error in ./spec/system/feedback_message_spec.rb:13:
invalid domain: ".localhost:3000"
RSpec::Retry: 3rd try ./spec/system/feedback_message_spec.rb:13
```

Reply:
Reference: https://github.com/forem/forem/actions/runs/5239228362/jobs/9458925719?pr=19603

Reply:
@maestromac , @klardotsh : I just found in `bin/setup` that .env_sample is used as base for Dev/Local environment.
```
  unless File.exist?(".env")
    FileUtils.cp ".env_sample", ".env"
  end
```

So, I need to find a different solution for test environment.
I can simply have a .env.test with a different APP_CONFIG or change `environments/test.rb` to not consider this.

I will do some more digging on this tomorrow.
If you have a preferred way to go, please let me know.

Reply:
Hi @heliocola — I'm not sure about this change (I'll defer to @maestromac and @klardotsh there) but the Ruby upgrade looks good without it, and this change seems unrelated. I'd suggest maybe moving this change to a separate PR, that way the Ruby upgrade can proceed on its own?

Reply:
I think splitting the .env for testing is the best way to go.
I will push that change and sync up this PR.

@maestromac : for development environment, this variable isn't used for cookie the way it is for Testing and Production.
Check these references: https://github.com/forem/forem/blob/main/config/initializers/middlewares.rb#L4C89-L4C89
But it still doesn't explain how this fails in the testing environment.

I will split the env file, and do some digging if I remember how the test env is affected.